### ISSUE TITLE
Capability to get probabilities for each tag in crf_decode

### ISSUE THREAD

**Issue by agarwalishan** [tensorflow/tensorflow#42178]
<em>Please make sure that this is a feature request. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:feature_template</em>

**System information**
- TensorFlow version (you are using): 1.15
- Are you willing to contribute it (Yes/No): Yes

**Describe the feature and the current behavior/state.**
I am talking about this specific repository.
[https://github.com/tensorflow/addons/blob/v0.10.0/tensorflow_addons/text/crf.py](url)

For now crf_decode function returns the tag indices with highest probability and a viterbi score for the whole sequence.
But the crf_decode do not returns the probabilities for every tag for each token in sequence like we get in softmax.
So I request to please add this capability because probabilities of tags help us know the confidence score and many more things.

**Will this change the current api? How?**
Yes, it will support more capabilities.

**Who will benefit with this feature?**
I believe many developers around the globe are looking for this feature.

**Any Other info.**
Please add it as soon as possible.

**Comment by amahendrakar** [comment]
@agarwalishan,
Could you please submit a new issue in the Addons repo using [this link](https://github.com/tensorflow/addons/issues/new/choose), so that we can track the issue there. Thanks!

**Comment by agarwalishan** [comment]
@amahendrakar,
Hey thanks for reply I have opened issue on the given link, you may close it here.
